features/session/logout.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Logout
+  As a member of the Berkeley Food Collective
+  I want to be able to click a logout button
+  So that I can logout of the application
+
+  Background: Database is loaded with test users
+    Given the following users in the database
+      | name             | email                     |
+      | Roberto Cardenas | robertocardenas@bsfc.coop |
+      | Andrew Mo        | andrewmo@bsfc.coop        |
+
+  When I start on the login page
+  When I fill in "session_email" with "robertocardenas@bsfc.coop"
+  And I press "Login"
+
+  Scenario: User wants to logout from the Announcements page
+    Given I should see "Announcements"
```

Comment:
this is minor but for readability, you should be more careful about the use of Given, When, and Then.  Given sets up preconditions, When simluates a user action, Then asserts postconditions.  in my view, for readability, line 13 above should be And, and `Then I should  see "Announcements"` seems like it should be part of the background steps (to confirm the login actually happened).  also that will help with the scenario of lines 16-19 currently not making sense: "Given i should see 'announcmenets' and I follow 'log out'" doesn't make sense as a story.  let me know if this comment makes sense, if not i can elaborate.